### ISSUE TITLE
new accounts should have previous forum posts marked read

### ISSUE THREAD

**Issue by cgolubi1** [buttonmen-dev/buttonmen#1909]
I think it's a little weird that when you create a new account, all posts on all forums are marked new.  You're not going to go back and read the entire forum --- that doesn't seem very helpful.

IMO it would be more helpful to go ahead and mark-read all existing forum posts at account creation time, so *active* threads would show up as unread and you could find them.

**Comment by AdmiralJota** [comment]
Purely to play devil's advocate:

Is there any chance that someone will want to manually mark the threads that they're not interested as read, so that they can keep track of which old threads they *are* interested in reading/skimming?

**Comment by irilyth** [comment]
Yeah, I think that might be true too -- a new player might well want to read *some* of the forum posts, and use the 'read' flag to keep track of what they've read and what they haven't. They couldn't use the 'new post' button to do it, but that's probably ok; that button's mostly useful for people who are already experienced players who want to keep up with the community.

Similarly, a new user may not want to bother with forum posts at all at first, and then some time later might want to get more involved in the community, at which point there will probably be a lot of posts piled up (since they joined). Maybe they want to read them all; maybe they don't.

One option to address both of these might be if you could say (a) mark all threads as read; and/or (b) mark all posts older than $N $time-units as read.
